src/treepoem/__init__.py

```diff
@@ -149,12 +149,17 @@ def generate_barcode(
     barcode_type: str,
     data: str | bytes,
     options: dict[str, str | bool] | None = None,
+    scale: int | None = 2,
```

Comment:
No need to allow `None`

Also let's make it keyword only, since we can for new parameters.